.github/workflows/sangertest.yml

```diff
@@ -1,23 +1,34 @@
 name: nf-core Sanger LSF tests
 
 on:
+  pull_request:
   workflow_dispatch:
 jobs:
   run-tower:
     name: Run LSF tests
     runs-on: ubuntu-latest
     steps:
+      - name: Sets env vars for pull_request
+        run: |
+          echo "REVISION=${{ github.sha }}" >> $GITHUB_ENV
```

Comment:
`github.event.pull_request.head.sha`
When PR, this variable should be used instead of ` github.sha `

Tower job failed again: https://tower.nf/orgs/sanger-tol/workspaces/github-ci/watch/3233P538jAUhKK